src/js/burials/containers/ConfirmationPage.jsx

```diff
@@ -15,29 +15,53 @@ const scrollToTop = () => {
 };
 
 class ConfirmationPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isExpanded: false };
+  }
+
   componentDidMount() {
-    focusElement('.schemaform-title > h1');
+    focusElement('.burial-page-title');
     scrollToTop();
   }
 
+  toggleExpanded = (e) => {
+    e.preventDefault();
+    this.setState({ isExpanded: !this.state.isExpanded });
+  }
+
   render() {
-    const time = this.props.form.submission.response.timestamp;
+    const form = this.props.form;
+    // const response = this.props.form.submission.response
+    //   ? this.props.form.submission.response.attributes
+    //   : {};
+    const name = form.data.claimantFullName;
 
     return (
-      <div>
-        <div className="usa-width-one-sixth medium-2 columns">
-          <i className="fa fa-check-circle hca-success-icon"></i>
-        </div>
-        <div className="usa-width-five-sixths medium-10 columns">
-          <h4 className="success-copy">You have successfully submitted your application for health care!</h4>
+      <div className="edu-benefits-submit-success">
+        <h3 className="burial-page-title">Claim received</h3>
+        <p>Normally processed within <strong>30 days</strong></p>
+        <p>
+          We may contact you for more information or documents.<br/>
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>Burial Benefit Claim <span className="additional">(Form 21P-530)</span></h4>
+          <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
```

Comment:
Does the `<span>` take out the extra space between first and last if there is no middle name? Looks like it in your screenshot, but I wanted to confirm.